### PR TITLE
Shared rubocop style

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-devtools.spec [hunk 1]
```diff
@@ -144,6 +144,7 @@ EOF
 %{_datadir}/YaST2/data/devtools/bin/y2makepot
 %{_datadir}/YaST2/data/devtools/bin/gettextdomains
 %{_datadir}/YaST2/data/devtools/bin/ycp_puttext
+%{_datadir}/YaST2/data/devtools/data/.rubocop_yast_style.yml
```
Comment (mvidner) [inline package/yast2-devtools.spec]: Without the leading dot please.

Comment (jreidinger) [inline package/yast2-devtools.spec]: good idea

**Comment by lslezak** [comment]
LGTM, thanks!

**Comment by lslezak** [comment]
I'm merging it so I could use it in the registration module...

**Pull request merged**